packages/phoenix-ng/projects/phoenix-ui-components/lib/components/ui-menu/overlay/overlay.component.ts

```diff
@@ -81,6 +85,9 @@ export class OverlayComponent implements AfterViewInit {
     if (width > this.MIN_RES_WIDTH && height > this.MIN_RES_HEIGHT) {
       overlayCardElement.style.width = width + 'px';
       overlayCardElement.style.height = height + 'px';
+      if (this.keepAspectRatioFixed) {
+        overlayCardElement.style.height = width / this.aspectRatio + 30 + 'px';
```

Comment:
If the boolean is true, this will set the height twice. You can improve the code here by using a ternary operator or `else`.

```ts
if (this.keepAspectRatioFixed) {
  overlayCardElement.style.height = width / this.aspectRatio + 30 + 'px';
} else {
  overlayCardElement.style.height = height + 'px';
}
```